app/views/welcome/index.html.haml

```diff
@@ -17,12 +17,14 @@
 
 .alert.alert-danger
   .p
-    Note that the API is in super early beta and is currently available for internal, experimental use only.
+    Note that the API is in beta, and is subject to change. Howeverm available for non-commercial production use.
```

Comment:
Why is the API still beta? I think we should say that the API is ready for production use, fix the typo in "however", or just remove it.